source/jormungandr/jormungandr/authentication.py

```diff
@@ -290,6 +289,12 @@ def get_user(token, abort_if_no_token=True):
         return g.user
 
 
+def get_unkown_user():
+    user = User(login="unknown_user")
+    user.id = 0
```

Comment:
Isn't it possible to do the following?

```suggestion
    user = User(id=0, login="unknown_user")
```

Reply:
The user [constructor](https://github.com/hove-io/navitia/blob/dev/source/navitiacommon/navitiacommon/models/__init__.py#L147) without id